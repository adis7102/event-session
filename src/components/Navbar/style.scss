@import "../../styles/colors";

.navbar-comp {
  width: 100%;
  padding: 41.5px 86px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-shadow: 0px 10px 5px -4px rgba(0,0,0,0.03);

  &-title {
    font-weight: 800;
    color: white;
    font-size: 55px;
    margin-left: 20px;
  }

  &-routing {
    display: flex;

    a {
      text-decoration: none;

      img {
        width: 16px;
        margin-right: 18px;
      }
    }

    .page-title {
      padding-left: 24px;
      border-left: 1px solid #bcc0d0;

      p {
        font-weight: 800;
        font-size: 18px;
        line-height: 24px;
      }
    }
  }

  
}
