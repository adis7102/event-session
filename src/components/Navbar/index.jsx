import React from "react";
import { Link } from 'react-router-dom';

import Arrow from "../../assets/icon-images/arrow.png";

import './style.scss';

const Navbar = () => {
  
  return (
    <div className="navbar-comp">
      <div className="navbar-comp-routing">
        <Link to="/">
          <img src={Arrow} alt="arrow" />
        </Link>
        <div className="page-title">
          <p>Event</p>
        </div>
      </div>
    </div>
  );
};

export default Navbar;
