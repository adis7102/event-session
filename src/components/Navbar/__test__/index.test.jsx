import React from "react";
import { BrowserRouter as Router, Route, Routes } from "react-router-dom";

import Navbar from "../index";

import { mount } from "enzyme";

it("should render Navbar correctly", () => {
  const wrapper = mount(
    <Router>
      <Navbar />
    </Router>
  );

  expect(wrapper).toMatchSnapshot();
});
