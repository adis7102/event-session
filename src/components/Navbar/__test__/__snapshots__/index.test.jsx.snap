// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Navbar correctly 1`] = `ReactWrapper {}`;
